articles/email/custom.md

```diff
@@ -89,19 +88,13 @@ function (user, context, callback) {
 }
 ```
 
-The rule only runs if the email is verified and if the email hasn't been sent before, to make sure it only runs once.
-
-Since the email will be sent by your own API you now have complete control over the verification email.
-
 ## Change Password Confirmation Email
 
-The Change Password functionality does not expose any extensiblity points that allow you to write custom code when users change their password. This means you'll just need to write some code after users changed their password.
-
-To handle this flow you'll need to host a "Change Password" form which captures the new password of the user and posts it to the [change password ticket endpoint](/api/v1#!#post--api-users--user_id--change_password_ticket). When calling this endpoint the password is not updated, but a "change password confirmation" link is generated.
+To handle password change requests, you will need to host a form to capture the user's new password and post it to the [change password ticket](/api/v1#!#post--api-users--user_id--change_password_ticket) endpoint. Calling this endpoint will generate a "Change Password Confirmation" link.
```

Comment:
Code might not be useful since people will use many different languages to implement, maybe we could do a flowchart?